src/constants/value_commit_v.rs

```diff
@@ -3701,8 +3701,8 @@ mod tests {
 
     #[test]
     fn generator() {
-        let hasher = pallas::Point::hash_to_curve(PERSONALIZATION);
-        let point = hasher(b"-v");
+        let hasher = pallas::Point::hash_to_curve(VALUE_COMMITMENT_PERSONALIZATION);
+        let point = hasher(b"v");
         let coords = point.to_affine().coordinates().unwrap();
 
         assert_eq!(*coords.x(), pallas::Base::from_bytes(&GENERATOR.0).unwrap());
```

Comment:
(Just below this.) Why do we include `NUM_WINDOWS` entries in the window table here, rather than just `NUM_WINDOWS_SHORT`?

Reply:
we don't need to, this has been removed. :+1: